src/CommunityToolkit.Maui.Core/Interfaces/IAvatarView.shared.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace CommunityToolkit.Maui.Core;
+
+/// <summary>Avatar view interface.</summary>
+public interface IAvatarView : IBorderView, ILabel, Microsoft.Maui.IImage, IImageSource
+{
+	#region Public Property
+
+	//// NOTE to implementer: Implement the following properties as public.
+
+	/// <summary>Gets a value indicating the avatar border colour.</summary>
+	Color BorderColor { get; }
+
+	/// <summary>Gets a value indicating the avatar border width.</summary>
+	double BorderWidth { get; }
+
+	/// <summary>Gets a value indicating the avatar corner radius (MAUI).</summary>
+	CornerRadius CornerRadius { get; }
+
+	/// <summary>Gets a value indicating the avatar padding.</summary>
+	Thickness AvatarPadding { get; }
+
+	#endregion Public Property
+
+	#region Explicit methods
+
+	//// NOTE to implementer: Implement these methods explicitly
+
+	/// <summary>Gets a value indicating the corner radius default.</summary>
+	CornerRadius CornerRadiusDefaultValue { get; }
```

Comment:
I think it should be in Public properties region. Do we need regions at all? As for me it’s a code smell

Reply:
@VladislavAntonyuk I copied the NOTES from Microsoft.Maui, and I added the regions to help differentiate.  I normally only ever use regions to segregate code when I'm working on it, but for production I tend to remove them.
I'll remove them.